fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayController.kt

```diff
@@ -442,7 +442,8 @@ class DefaultTabsTrayController(
         handleTabDeletion(tab.id, TrayPagerAdapter.INACTIVE_TABS_FEATURE_NAME)
     }
 
-    override fun handleInactiveTabsHeaderClicked(expanded: Boolean) {
+    override fun handleInactiveTabsHeaderClicked() {
+        val expanded = !appStore.state.inactiveTabsExpanded
```

Comment:
I am not sure I agree with this change. I think we could better match the intent of a unidirectional data flow by either
1. Taking this change a step further and moving this logic into the `AppStoreReducer`
2. Leaving things as they currently are